ci/check-manifest.sh

```diff
@@ -32,6 +32,8 @@ YAMLS=(
     "build/yamls/antrea-gke.yml"
     "build/yamls/antrea-aks.yml"
     "build/yamls/antrea-windows.yml"
+    "build/yamls/antrea-windows-containerd.yml"
```

Comment:
I found a new MC manifest is also missed in this file. I submitted a PR here https://github.com/antrea-io/antrea/pull/5672 to fix it, could you check if possible to use the same pattern to avoid the same case happen in the future?

Reply:
Sure, updated.